AppCenter/AppCenter/Internals/Util/MSUtility.m

```diff
@@ -40,15 +40,23 @@ + (NSString *)sdkVersion {
 }
 
 + (NSObject *)unarchiveKeyedData:(NSData *)data {
```

Comment:
There are usages of this function that wrapped in `@try`/`@catch` we need to clean it and check if result is `nil` instead.